### PR TITLE
Minor update to the ISSUE/PR template

### PR DESCRIPTION
**Pull request by WestleyK**
## Description of the Pull Request (PR):

This PR updates the issue/pr template, this has not changed since ~ singularity v2.3. Its probably time for a little update.

Also, it may be worth adding bug/feature issue template in the near future -> https://github.com/sylabs/singularity/issues/3763

### This fixes or addresses the following GitHub issues:

 - Fixes #

<br>

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 
-**This fixes or addresses the following GitHub issues:**
+### This fixes or addresses the following GitHub issues:
 
-- Fixes #
+ - Fixes #
 
+<br>
 
 **Before submitting a PR, make sure you have done the following:**
```
Comment (sashayakovtseva) [inline .github/PULL_REQUEST_TEMPLATE.md]: Why not changing `*` here either?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
 fixing or enhancing.
 
 
-**This fixes or addresses the following GitHub issues:**
+### This fixes or addresses the following GitHub issues:
 
-- Fixes #
+ - Fixes #
 
+<br>
```
Comment (sashayakovtseva) [inline .github/PULL_REQUEST_TEMPLATE.md]: Is that necessary?

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -1,18 +1,32 @@
...
-Write here.
+What actually happend? Why was it incorrect?
+
 
 ### Steps to reproduce behavior
 
-Write here.
+Please fill out as much information as possible.
+
+#### What OS are you running
```
Comment (sashayakovtseva) [inline .github/ISSUE_TEMPLATE.md]: I suggest to have a single environment section like [here](https://github.com/sylabs/singularity-cri/blob/master/.github/ISSUE_TEMPLATE/bug_report.md#environment).

And remove `Version of Singularity`, `What OS are you running` and `How did you install Singularity`.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (WestleyK) [reply .github/PULL_REQUEST_TEMPLATE.md]: Nope. Removed.

**Review by mem — APPROVED** [review]

File: .github/ISSUE_TEMPLATE.md (hunk 3 above)
Comment (WestleyK) [reply .github/ISSUE_TEMPLATE.md]: I added a `Environment` section like suggested, but I think we should keep the `Version of Singularity` section. Other sections can be added/removed later if we find out that they are not useful.

**Comment by WestleyK** [comment]
Just removed the unnecessary merge commits, nothing else changed while forced pushed.

**Review by sashayakovtseva — APPROVED** [review]

**Pull request merged**